shared/checks/oval/installed_env_has_ovirt-host_package.xml

```diff
@@ -0,0 +1,38 @@
+<def-group>
+
+  <definition class="inventory"
+  id="installed_env_has_ovirt-host_package" version="1">
+    <metadata>
+      <title>Package ovirt-host is installed</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
+      <description>Checks if package ovirt-host is installed.</description>
+      <reference ref_id="cpe:/a:ovirt-host" source="CPE" />
+    </metadata>
+    <criteria>
+      <criterion comment="Package ovirt-host is installed" test_ref="test_env_has_ovirt-host_installed" />
+    </criteria>
+  </definition>
+
+{{% if pkg_system == "rpm" %}}
+  <linux:rpminfo_test check="all" check_existence="at_least_one_exists"
+  id="test_env_has_ovirt-host_installed" version="1"
+  comment="system has package ovirt-host installed">
+    <linux:object object_ref="obj_env_has_ovirt-host_installed" />
+  </linux:rpminfo_test>
+  <linux:rpminfo_object id="obj_env_has_ovirt-host_installed" version="1">
+    <linux:name>ovirt-host</linux:name>
```

Comment:
Can this rule check multiple packages? Sorry I did not emphasize this enough, but the profile should apply to both hosts and manager, that would mean having installed `ovirt-host OR ovirt-engine`. Does it make sense?

Reply:
> Can this rule check multiple packages? 

Yes, it is possible to check for multiple packages.

> Sorry I did not emphasize this enough, but the profile should apply to both hosts and manager, that would mean having installed `ovirt-host OR ovirt-engine`. Does it make sense?

It makes sense, but then the question I have is if the hosts and the manager will always have the same restrictions. Could a rule be ok for the manager but not for the host? Or vice-versa?

For example, does the system acting as the manager also need to allow for `root` login?

I guess one complicator is that the manager can be self-hosted right?
In this case a single system acts both as the manager and the host.
So it should be simpler to extend the platform to identify both host and manager.

Reply:
> It makes sense, but then the question I have is if the hosts and the manager will always have the same restrictions. Could a rule be ok for the manager but not for the host? Or vice-versa?

It should not hurt if the rule applies for both.

> For example, does the system acting as the manager also need to allow for root login? 

Regular manager does not, but as you have mentioned the HE needs it. 

> I guess one complicator is that the manager can be self-hosted right?
In this case a single system acts both as the manager and the host.
So it should be simpler to extend the platform to identify both host and manager.

We have counted 6 or 7 rules that will be excluded (the 7th is not yet decided if we can fix it). And ~half of them is common for both I am not sure if it makes sense to split it. If it does I won't have anything against that.

Reply:
One problem of having the same restrictions for both the manager and hosts is that they will not be as much aligned/hardened to the STIG standard as they could be. It could end up also misleading sys admins and auditors.

On a second thought, it is probably not that much complicated to distinguish rules that don't work for the manager, don't work for the hosts, or don't work for both.

On the self-hosted engine case (HE), are both `ovirt-host` and `ovirt-engine` installed on the machine?

Reply:
No HE has only ovirt-host installed because it as a VM running on the first host.

Reply:
So I guess the Host running the virtualized Manager needs the some kind of "freedom" as the Manager?

Anyway, we can go with simpler approach and improve if necessary.

Reply:
Except some installation steps the host is not that special. But I agree, let's have common exclude for both.